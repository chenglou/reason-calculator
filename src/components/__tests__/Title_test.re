open Jest;
open Expect;

let _ =

describe "Title" (fun _ => {
  test "renders" (fun _ => {
    let tree = Title.createElement children::[] () |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });
});
