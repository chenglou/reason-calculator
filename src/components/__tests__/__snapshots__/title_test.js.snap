// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders 1`] = `
<h1
  className={
    Object {
      "data-css-tc8z0v": "",
    }
  }
>
  Reason Calculator
</h1>
`;
